README.rst

```diff
@@ -2,10 +2,55 @@
 LSST the Docs: Mason
 ####################
 
-**Mason** is the document build tool for LSST software documentation that runs in the Jenkins build environment.
+**Mason** is the build tool for LSST software documentation that runs in the Jenkins build environment.
 It is a part of the **LSST the Docs** (LTD) service for continuous documentation deployment.
 You can learn more about LTD in our `SQR-006`_ technote.
 
+Installation
+============
+
+Mason is intended to work with either Python 2.7, 3.4, or 3.5.
+It *must* run from the same Python as the software being documented, however.
+
+::
+
```

Comment:
So this is an interesting point because ltd-mason is meant to specifically be installed in the same environment that the Eups product (e.g., lsst_apps) was built in.

@timj do you know the best way to have a Python virtual environment (conda environment or otherwise) layered on top of an lsstsw install?

Is the best way to create an environment, have lsstsw build in that, and then also install/run ltd-mason in that environment? Alternatively I know that virtual environments can incorporate site-packages/, but I don't know mechanism also incorporates packages that weren't actually pip/setup.py install'd into site-packages.